### PR TITLE
Add Player::ShowHitMarker

### PR DESCRIPTION
**Pull request by green1052**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1366,6 +1366,11 @@ public void ShowHint(string message, float duration = 3f)
             HintDisplay.Show(new TextHint(message, parameters, null, duration));
         }
 
+        /// <summary>
+        /// Show HitMarker.
```
Comment (PintTheDragon) [inline Exiled.API/Features/Player.cs:1370]: Change to `Show a HitMarker`.

File: Exiled.API/Features/Round.cs [hunk 2]
```diff
@@ -26,6 +26,11 @@ public static class Round
         /// </summary>
         public static DateTime StartedTime => DateTime.Now - ElapsedTime;
 
+        /// <summary>
+        /// Gets the round progress time.
+        /// </summary>
+        public static TimeSpan ProgressTime => DateTime.Now.Subtract(StartedTime);
+
```
Comment (PintTheDragon) [inline Exiled.API/Features/Round.cs:33]: What does this value actually mean? `Now - (Now - ElapsedTime)` will just end up being `ElapsedTime`, so this property seems redundant.

**Review by PintTheDragon — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Player.cs [hunk 3]
```diff
@@ -1366,6 +1366,11 @@ public void ShowHint(string message, float duration = 3f)
...
+        /// </summary>
+        public void ShowHitMarker() => ReferenceHub.GetComponent<Scp173PlayerScript>()?.TargetHitMarker(Connection);
```
Comment (Nekonyx) [inline Exiled.API/Features/Player.cs:1372]: `ReferenceHub.characterClassManager.Scp173`?

**Review by iopietro — APPROVED** [review]

File: Exiled.API/Features/Player.cs [hunk 4]
```diff
@@ -1366,6 +1366,11 @@ public void ShowHint(string message, float duration = 3f)
             HintDisplay.Show(new TextHint(message, parameters, null, duration));
         }
 
+        /// <summary>
+        /// Show a HitMarker.
```
Comment (iRebbok) [inline Exiled.API/Features/Player.cs:1370]:
```suggestion
        /// Shows a HitMarker.
```

as following most of the documentation.

**Review by iRebbok — CHANGES_REQUESTED** [review]

**Review by BuildBoy12 — APPROVED** [review]

**Review by iRebbok — APPROVED** [review]

**Review by BuildBoy12 — APPROVED** [review]

**Pull request merged**